recipes/serdepp/all/conanfile.py

```diff
@@ -52,48 +49,62 @@ def _compilers_minimum_version(self):
             "apple-clang": "10",
         }
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("nameof/0.10.1")
+        self.requires("magic_enum/0.7.3")
+        if self.options.with_toml11:
+            self.requires("toml11/3.7.0")
+        if self.options.with_yamlcpp:
+            self.requires("yaml-cpp/0.7.0")
+        if self.options.with_rapidjson:
+            self.requires("rapidjson/1.1.0")
+        if self.options.with_fmt:
+            self.requires("fmt/8.1.1")
```

Comment:
Check the bump demps custom command in the extensions repo at https://github.com/conan-io/conan-extensions/tree/main/extensions/commands/recipe which might help you bumping versions of the recipes to change. I'd like for this to be bumped, but can be done in a follow-up PR @valgur